### ISSUE TITLE
TypeError in import_mock() in tests

### ISSUE THREAD

**Issue by mreficent** [OCA/openupgradelib#164]
Please, can somebody fix this? Example in https://travis-ci.org/OCA/openupgradelib/builds/565402166.
```yml
Traceback (most recent call last):
  File "setup.py", line 56, in <module>
    tests_require=test_requirements
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/setuptools/__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "/opt/python/2.7.15/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/opt/python/2.7.15/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/opt/python/2.7.15/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/setuptools/command/test.py", line 228, in run
    self.run_tests()
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/setuptools/command/test.py", line 250, in run_tests
    exit=False,
  File "/opt/python/2.7.15/lib/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/opt/python/2.7.15/lib/python2.7/unittest/main.py", line 149, in parseArgs
    self.createTests()
  File "/opt/python/2.7.15/lib/python2.7/unittest/main.py", line 158, in createTests
    self.module)
  File "/opt/python/2.7.15/lib/python2.7/unittest/loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/opt/python/2.7.15/lib/python2.7/unittest/loader.py", line 91, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
  File "/home/travis/build/OCA/openupgradelib/tests/__init__.py", line 2, in <module>
    from . import test_openupgradelib  # noqa: F401
  File "/home/travis/build/OCA/openupgradelib/tests/test_openupgradelib.py", line 44, in <module>
    from openupgradelib import openupgrade
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1062, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1128, in _mock_call
    ret_val = effect(*args, **kwargs)
  File "/home/travis/build/OCA/openupgradelib/tests/test_openupgradelib.py", line 26, in import_mock
    return orig_import(name, *args)
  File "/home/travis/build/OCA/openupgradelib/openupgradelib/openupgrade.py", line 57, in <module>
    from lxml import etree
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1062, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1128, in _mock_call
    ret_val = effect(*args, **kwargs)
  File "/home/travis/build/OCA/openupgradelib/tests/test_openupgradelib.py", line 26, in import_mock
    return orig_import(name, *args)
  File "src/lxml/serializer.pxi", line 622, in init lxml.etree
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/codecs.py", line 1011, in getwriter
    return lookup(encoding).streamwriter
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/encodings/__init__.py", line 100, in search_function
    level=0)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1062, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/mock/mock.py", line 1128, in _mock_call
    ret_val = effect(*args, **kwargs)
TypeError: import_mock() got an unexpected keyword argument 'fromlist'
The command "coverage run setup.py test" exited with 1.
```

**Comment by pedrobaeza** [comment]
Is this happening in main branch also?

**Comment by mreficent** [comment]
Yes, you can see in last commit, in https://travis-ci.org/OCA/openupgradelib/builds/563888791.

The problem has raised in the update of travis you did yesterday.

**Comment by pedrobaeza** [comment]
Update of Travis? Do you mean MQT?

**Comment by mreficent** [comment]
Yes, I dunno. Something happened yesterday that affected this.

**Comment by mreficent** [comment]
Fixed in https://github.com/OCA/openupgradelib/pull/165.

**Comment by Yajo** [comment]
This can be closed.

BTW @mreficent remember that if you write in your PR description "Fix #164" Instead of "Avoid #164", Github autodetects that special word and merging that PR in the repo main branch will close the issue automatically.
